modules/publicize/ui.php

```diff
@@ -22,8 +22,9 @@ function __construct() {
 	}
 
 	function init() {
-		// Show only to users with the capability required to manage their Publicize connections.
-		if ( ! current_user_can( 'publish_posts' ) ) {
+		// Show only to users with the capability required to manage their Publicize connections.  Filter to allow for custom roles. 
+		$capability = apply_filters( 'publicize_capability', 'publish_posts' );
```

Comment:
Also, let's make the filter name `jetpack_publicize_capability` for better safety 👍